docs/dast_runbook.md

```diff
@@ -0,0 +1,29 @@
+# DAST SCAN
+
+Dynamic Application Security Testing
+([DAST](https://splunk.atlassian.net/wiki/spaces/SEC/pages/1078262957897/Centralized+Vulnerability+Management+on+JIRA#DAST))
```

Comment:
I think these docs aren't publicly available, whereas UCC docs are publicly open. Is there any other way we can document this?

Reply:
@kdoroszko-splunk let's move it to our UCC team folder in Google Drive.

Reply:
Ok, I will do that.